src/WordPress/Blueprints/BlueprintParser.php

```diff
@@ -152,14 +153,15 @@ private function findSubErrorForSpecificAnyOfOption( \Swaggest\JsonSchema\Except
 			}
 		}
 		if ( ! $e->subErrors ) {
-			return;
+			return $e->error;
```

Comment:
I'm not sure if that's right, but this function isn't used at the moment so you could comment it out as well. We'll soon need some version of it that can work with the Opis JSON Schema validation library (this version works with the no-longer-used Swaggest library).

Reply:
I am commenting out the unused method and leaving several TODOs hinting at things to consider.